utils/rpms/daos.spec

```diff
@@ -110,6 +110,14 @@ BuildRequires: systemd-rpm-macros
 %endif
 BuildRequires: libuuid-devel
 
+# Needed for debugging tasks
+%if (0%{?rhel} >= 8)
+BuildRequires: libasan
```

Comment:
Does it mean that we have to always install `libasan` for RPM build?

Reply:
For sure is not needed if you are not instrumenting your binary with the Asan lib.
However, as for valgrind, it should not introduce some big dependencies constraints.
https://github.com/daos-stack/daos/blob/26c2219125f6330c1404461fcc78c0525522ec99/utils/rpms/daos.spec#L79
From my side I do think it is worth of it to try to make it conditional if it is compiled or not with the Asan lib.